walkthroughs/howto-k8s-mtls-sds-based/README.md

```diff
@@ -45,7 +45,7 @@ v1.3.0
 
     export ENVOY_IMAGE=...
     
-    **Note:** By default, the appmesh-controller v1.4.0 comes with envoy version 1.17.2.0. However, this walkthrough is designed to work with envoy version >= 1.15.1.0 & <= 1.16.3.0 and so we recommend you to pick an envoy version in this range.
+    **Note:** This walkthrough is designed to work with envoy version >= 1.15.1.0 & <= 1.16.3.0 and so we recommend you to pick an envoy version in this range if the default envoy version doesn't fall in this range. 
```

Comment:
Sentence is too long. Can it be just "This walkthrough requires using Envoy versions >= 1.15.1.0 and <= 1.16.3.0" ?

Reply:
Yes, also we do have latest envoy with spire support now. So will update this PR